transaction-pool/Cargo.toml

```diff
@@ -13,4 +13,4 @@ smallvec = "0.6"
 trace-time = { path = "../trace-time", version = "0.1" }
 
 [dev-dependencies]
-ethereum-types = { version = "0.6", path = "../ethereum-types" }
+ethereum-types = { version = "0.7", path = "../ethereum-types" }
```

Comment:
Bump `transaction-pool` version too.

Reply:
My rational was that it is only dev-deps, but yes better safe than sorry. Minor bump should be enough for this case?

Reply:
Oh damn, missed that, apologies. No, that's fine: as long as tests pass with the new version it's fine to keep unchanged.